### ISSUE TITLE
pagination=False for select_multiple is not working

### ISSUE THREAD

**Issue by anentropic** [petereon/beaupy#62]
Currently if `len(options) > page_size` then the select list doesn't work properly - all items are shown but checks are misattributed to different options and some options are not selectable

`pagination=False` is the default, my code was like:

```python
        indices = select_multiple(
            options=options,
            ticked_indices=ticked_indices,
            cursor_index=ticked_indices[0],
            minimal_count=1,
            return_indices=True,
        )
```

(default page size is 5, I had 8 options)

I was able to work around it by setting `page_size=len(options),`

**Comment by petereon** [comment]
Hi @anentropic, thanks for reporting the issue. I will investigate!

**Comment by anentropic** [comment]
I don't think I've pressed the `END` key

Try with just `ticked_indices = [5]`

What I would see is the first option was ticked (I guess because index `5` would be first item of second page if paginated)

And then pressing space bar on items 6, 7 or 8 (index 5, 6, 7) would tick options 1, 2 or 3 (presumably index 0, 1, 2 of second page)

**Comment by petereon** [comment]
Released as [`3.5.4`](https://github.com/petereon/beaupy/releases/tag/v3.5.4) also available on [PyPI](https://pypi.org/project/beaupy/3.5.4/)

**Comment by anentropic** [comment]
Thanks, working for me now 👍
